basecli/exp.go

```diff
@@ -16,64 +16,77 @@ limitations under the License.
 package basecli
 
 import (
+	"fmt"
 	"io/ioutil"
+	"path"
 
-	"github.com/iter8-tools/iter8/base"
 	"github.com/iter8-tools/iter8/base/log"
 	"github.com/spf13/cobra"
+	"helm.sh/helm/v3/pkg/chart"
+	"helm.sh/helm/v3/pkg/chart/loader"
 	"helm.sh/helm/v3/pkg/chartutil"
-)
-
-const (
-	// Path to experiment template file
-	expTemplatePath = "experiment.tpl"
+	"helm.sh/helm/v3/pkg/cli"
+	"helm.sh/helm/v3/pkg/engine"
+	"helm.sh/helm/v3/pkg/getter"
 )
 
 // expCmd represents the exp command
 var expCmd = &cobra.Command{
 	Use:   "exp",
-	Short: "Render experiment template in the file experiment.tpl with values",
+	Short: "Render experiment.yaml file for local experiments",
```

Comment:
why is this only for  *local* experiments?

Reply:
Fixed.